app/components/Card.tsx

```diff
@@ -1,19 +1,22 @@
 import { tx } from '@helpwave/common/twind/index'
 import type { PropsWithChildren } from 'react'
+import type { Class } from '@twind/core'
 
 export type CardProps = {
   isSelected?: boolean,
-  onTileClicked?: () => void
+  onTileClicked?: () => void,
```

Comment:
I made a mistake in the last (2) review(s): should be `onTileClick` instead of `onTileClicked`

Sorry, was tired 😅